daprdocs/content/en/developing-applications/integrations/open-service-mesh.md

```diff
@@ -1,31 +1,25 @@
 ---
 type: docs
-title: "Running Dapr and Open Service Mesh together"
-linkTitle: "Open Service Mesh"
+title: "How to: Run Dapr and Open Service Mesh together"
```

Comment:
I suggest that we remove this article entirely from the docs. MS has decide to stop working on OSM https://openservicemesh.io/blog/osm-project-update/ and now tell people to use Istio. No point in keeping this.